pyproject.toml

```diff
@@ -0,0 +1,10 @@
+[build-system]
+requires = [
+    "numpy==1.17; python_version=='3.5'",
+    "numpy==1.17; python_version=='3.6'",
+    "numpy==1.17; python_version=='3.7'",
+    "numpy==1.17; python_version=='3.8'",
+    "numpy==1.19; python_version=='3.9'",
```

Comment:
note: numpy only supports version 1.19 and higher for python 3.9+. see https://github.com/scipy/oldest-supported-numpy/blob/master/setup.cfg